### PR TITLE
Put an explanatory comment in trigger files that exist

### PR DESCRIPTION
**Pull request by kennknowles**
I noticed these files that got merged with PRs. We ought to be more conscientious! But they could serve a purpose. Rather than deleting them, we can add a little comment that says how to use them. Yes it is redundant to put it in every file but probably nice for each one to just say what it is for rather than cause you to look around.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [x] Update `CHANGES.md` with noteworthy changes.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by kennknowles** [comment]
@Abacn @damccorm

**Review by damccorm — APPROVED** [review]
Thanks, I like it. Probably gonna fail some test suites on this PR now though :)

**Comment by Abacn** [comment]
PostCommit Java ValidatesRunner SparkStructuredStreaming failing is fixed in #30589 -- may need another cherry-pick into release branch to get this test green also

File: .github/trigger_files/beam_PostCommit_Java_Avro_Versions.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "comment": "Modify this file in a trivial way to cause this test suite to run"
```
[the remark this answers is not in the document]
Comment (kennknowles) [reply .github/trigger_files/beam_PostCommit_Java_Avro_Versions.json:2]: Ha, it was me :-)

What I had in mind was adding/removing whitespace. So the file would always be there as a reminder of what the options are and so you don't have to go look at README.

[the remark this answers is not in the document]
Comment (Abacn) [reply .github/trigger_files/beam_PostCommit_Java_Avro_Versions.json:2]: yeah this makes sense

**Comment by kennknowles** [comment]
I'm being pedantic about actually waiting for green here, but TBH it is a waste of time and resources. Perhaps we just go for it and merge?

**Comment by damccorm** [comment]
> I'm being pedantic about actually waiting for green here, but TBH it is a waste of time and resources. Perhaps we just go for it and merge?

SGTM

**Pull request merged**